examples/plot_colormap_components.py

```diff
@@ -0,0 +1,12 @@
+# -*- coding: utf-8 -*-
+"""
+Plot colormap components
+========================
+
+Quickly plot the RGB components of a colormap.
+"""
+
+import WrightTools as wt
+
+cmap = wt.artists.colormaps['default']
+wt.artists.plot_colormap_components(cmap)
```

Comment:
Would it be useful to have a comparison of several colormaps on the same page?

Reply:
probably but it's not within the scope right now

note that we have a figure with all of the cmaps on the artists page